File: C_PROGRAMMING/C_PROGRAMS/Statements/0_Basics/program9.c

// WAP take a number and check whether it is positive or negative..
//
#include<stdio.h>
void main(){
  int num;

  printf("\n Enter the number:");
  scanf("%d",&num);
  
  if(num==0)
  {
    printf("\n%d you entered number is zero.\n",num);
  }
  else if(num<1){
    printf("\n%d is Negative number\n",num);
   }
  else{
    printf("\n%d is Positive number\n",num);
  }

}

File: JAVA_PROGRAMMING/ASSIGNMENTS/CONTROL_STATEMENTS/Pattern/Assignment_1/program9.java

// WAP to print following pattern

// 	1C3     4B2     9A1
// 	16C3    25B2    36A1
// 	49C3    64B2    81A1


class pattern{

	public static void main(String[] args){
		int num = 1;

		for(int i = 1; i <= 3;i++){
	
			char ch = 'C';
			int n = 3;

			for(int j = 1; j <= 3; j++){

				System.out.print((num*num) +""+ch+""+n+"\t");
				num++;
				ch--;
				n--;
			}
		
		System.out.println();
		}
	}
}
